src/components/Pagination/Pagination.js

```diff
@@ -126,22 +142,38 @@ Pagination.propTypes = {
    */
   pagesToShow: PropTypes.number,
   /**
-   * Label to be used on buttomn of next
+   * Label to be used on button of next
    */
   nextLabel: PropTypes.string,
   /**
-   * Label to be used on buttomn of previous
+   * Label to be used on button of previous
+   */
+  previousLabel: PropTypes.string,
+  /**
+   * Text to be shown on the pagination footer
+   */
+  footer: PropTypes.string,
+  /**
+   * Align the pagination.
+   */
+  align: PropTypes.string,
```

Comment:
What are the options here, besides `center`? I'd prefer something like:

```js
  PropTypes.oneOf([
    'center',
    'left',
    'right'
  ])
```

Same for the `justify` prop type.

Reply:
There are predefined CSS options for this. I left it open for text cause I can't be sure whether future CSS versions will add more options. We can always add them, but this means stricter maintenance. I don't have a hard opinion on this however.

Reply:
I'm not familiar with the design for this component. Do you want to allow _all_ alignment options or just a subset? I wonder if this is better handled by passing in a custom classname. 🤔

Reply:
Right now I'm passing the rest of the props down to the `Container`, so probably that can be used to pass custom styles. Haven't tested whether it will work though.
About allowing all options, that I think is a very good question. I don't think all options are necessary to be allowed.
So what do you think, should I go for passing the classname or limiting the options?

Reply:
I think the classname option is what we use elsewhere in Circuit UI, so I'd stick to it for consistency sake.

Reply:
It's odd, I agree :) Removed the props.